src/main/java/seedu/address/model/userimage/UserImage.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.userimage;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Objects;
+import javax.imageio.ImageIO;
+
+/**
+ * Creates a UserImage object that contains an image file with description
+ */
```

Comment:
Same as above

Reply:
Thanks for the reminder. Javadoc has been changed to make it clearer
```
/**
 * Represents a Person's associated image in the address book.
 * Guarantees: immutable; is valid as declared in {@link #isImage(FilePath)}
 */
```